scripts/cleanup.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+#
+# Exit on error. Append "|| true" if you expect an error.
+set -o errexit
+# Exit on error inside any functions or subshells.
+set -o errtrace
+# Do not allow use of undefined vars. Use ${VAR:-} to use an undefined VAR
+set -o nounset
+# Catch the error in case mysqldump fails (but gzip succeeds) in `mysqldump |gzip`
+set -o pipefail
+
+
+## @brief list all virtual machines running dockers
+function cleanMachines(){
+
+    IFS=', ' read -r -a machines <<< $(docker-machine ls | awk '{print $1}' | tail -n +2 | tr '\n' ', ')
+
+    for machine_name in $machines
```

Comment:
since you are using an array here, you will need to do ${machines[@]} to loop through all the entries